From 9a8b8b875881bd8386a1bb1443592722a653aacb Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Fri, 24 Jul 2020 10:38:54 +0800
Subject: [PATCH] Test case for Driver - MpEqu.inc file

Description: Change the value of 'ModeHighSegmentLocation' in the UefiCpuPkg/Library/MpInitLib/X64/MpEqu.inc

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 UefiCpuPkg/Library/MpInitLib/X64/MpEqu.inc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/UefiCpuPkg/Library/MpInitLib/X64/MpEqu.inc b/UefiCpuPkg/Library/MpInitLib/X64/MpEqu.inc
index 58ef369342..c94d860bc6 100644
--- a/UefiCpuPkg/Library/MpInitLib/X64/MpEqu.inc
+++ b/UefiCpuPkg/Library/MpInitLib/X64/MpEqu.inc
@@ -39,5 +39,5 @@ InitializeFloatingPointUnitsAddress equ  LockLocation + 8Ch
 ModeTransitionMemoryLocation        equ  LockLocation + 94h
 ModeTransitionSegmentLocation       equ  LockLocation + 98h
 ModeHighMemoryLocation              equ  LockLocation + 9Ah
-ModeHighSegmentLocation             equ  LockLocation + 9Eh
+ModeHighSegmentLocation             equ  LockLocation + 9Ch
 Enable5LevelPagingLocation          equ  LockLocation + 0A0h
-- 
2.27.0.windows.1

